lib/CacheMapper.ts

```diff
@@ -0,0 +1,163 @@
+export class CacheMapper<K, V> implements Map<K, V> {
+  private _innerMap: Map<K, V>;
+  size: number;
```

Comment:
We should make `size` a read-only getter that returns `this._innerMap.size`.

Reply:
when removed all size manipulator, will return undefined
![image](https://github.com/hexojs/hexo-util/assets/12471057/14ae9b6b-8541-41a5-a50e-44476baf1e7d)

Reply:
or using keys counter, this will works like charm
![image](https://github.com/hexojs/hexo-util/assets/12471057/d2893705-07fb-4570-a440-f04bd2d7feaa)